libraries/SITL/SITL.cpp

```diff
@@ -106,6 +126,11 @@ const AP_Param::GroupInfo SIM::var_info2[] = {
 
     AP_GROUPINFO("WIND_DIR_Z",  10, SIM,  wind_dir_z,     0),
     AP_GROUPINFO("WIND_T"      ,15, SIM,  wind_type, SIM::WIND_TYPE_SQRT),
+    // @Param: WIND_T_ALT
+    // @DisplayName: Full Wind Altitude
+    // @Description: Altitude at which wind reaches full strength, rising from 0 at ground level
```

Comment:
Might be nice to specify the wind/altitude model here

Reply:
probably others also...but its a start